Deprecate checkmax/max_params docs (#2739)
nf-core/website: sites/docs/src/content/docs/tutorials/use_nf-core_pipelines/writing_institutional_profiles.md

Note that for the `config_profile_contact`, it is best to indicate a specific person. This will typically be someone who wrote the config (via their name & github handle) or whoever will maintain it at the institution (e.g. email of IT Department, Institution X), i.e. someone who can be contacted if there are questions or problems and how to contact them.

Finally, if you have a common resource directory for the AWS `iGenomes` collection of reference genomes, this can can also go in the `params` scope.

```nextflow
params {
  config_profile_description = '<your_cluster_name> cluster profile provided by nf-core/configs.'
  config_profile_contact = '<your_name> (<your_github_handle>)'
  config_profile_url = 'https://<institutional_url>.com'
  igenomes_base = '/<path>/<to>/igenomes/'
}
```

:::note{collapse title="Note on older nf-core pipelines"}

> If using older pipelines, you will also need to specify the `max_*` series of params (now not used in the pipeline template).
> 
> These are used by nf-core pipelines to limit automatic resubmission of resource-related failed jobs to ensure submitted retries do not exceed the maximum available on your cluster.
> These values should be the ones you found for the largest node of your cluster (i.e., the largest node a user's job can be submitted to). For example:

Reviewer: ```suggestion
If using older pipelines, you will also need to specify the `max_*` series of params (now not used in the pipeline template).

These are used by nf-core pipelines to limit the automatic resubmission of resource-related failed jobs to ensure submitted retries do not exceed the maximum available on your cluster.
These values should be the ones you found for the largest node of your cluster (i.e., the largest node a user's job can be submitted to). For example:
```

  max_memory = 2.TB
  max_cpus = 128
  max_time = 720.h
}
```

:::

#### process scope

> Next, we can use the `process` scope to define certain resource specifications, scheduler to use, and associated options. Any option specified in this scope means that all processes in a pipeline will use the settings defined here.

Reviewer: ```suggestion
Next, we can use the `process` scope to define certain resource specifications, schedulers to use, and associated options. Any option specified in this scope means that all processes in a pipeline will use the settings defined here.
```

> First, you should specify a `resourceLimits` list based on resource maximums of your machine or infrastructure.

Reviewer: ```suggestion
First, you should specify a `resourceLimits` list based on the maximum resources available on your machine or infrastructure.
```

> These values are used by nf-core pipelines to limit automatic resubmission of resource-related failed jobs to ensure submitted retries do not exceed the maximum available on your cluster.

Reviewer: ```suggestion
These values are used by nf-core pipelines to limit the automatic resubmission of resource-related failed jobs to ensure submitted retries do not exceed the maximum available on your cluster.
```

> These values should be the ones you found for the largest node of your cluster (i.e., the largest node a user's job can be submitted to).

Reviewer: ```suggestion
These values should be the ones you found for the largest node of your cluster (i.e., the largest node to which a user's job can be submitted).
```
